src/debug/ee/functioninfo.cpp

```diff
@@ -2332,31 +2361,9 @@ DebuggerJitInfo *DebuggerJitInfo::GetJitInfoByAddress(const BYTE *pbAddr )
     return dji;
 }
 
-PTR_DebuggerJitInfo DebuggerMethodInfo::GetLatestJitInfo(MethodDesc *mdesc)
+DebuggerJitInfo *DebuggerMethodInfo::GetLatestJitInfo()
 {
-    // dac checks ngen'ed image content first, so
```

Comment:
As part of leaving GetJitInfoWorker alone, presumably this would revert too.

Reply:
Yep will revert